storage/cloud-client/storage_get_bucket_metadata.py

```diff
@@ -39,6 +39,7 @@ def bucket_metadata(bucket_name):
     )
     print("Default KMS Key Name: {}".format(bucket.default_kms_key_name))
```

Comment:
Please use f-strings.

Reply:
These samples still need to work with Py 2.7 right? I thought that ruled out fstrings.

Reply:
AFAIK, App Engine Standard is the only product that is still required to work with 2.7.    These samples do not run tests on 2.7.